Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:20
+
+WORKDIR /src/app
+
+COPY package*.json .
+
+RUN npm install
+
+COPY . .
+
+RUN npm run build
```

Comment:
No need to run build every time docker runs. You can use nodemon to listen to necessary files. That's a better way to do hot reloading.
```suggestion
```

Add a nodemon.json to the root of your project and add this.

```json
{
  "verbose": true,
  "watch": ["src/**/*.ts", "src/**/*.yaml"],
  "execMap": {
    "ts": "node --inspect=0.0.0.0:9229 --nolazy -r ts-node/register"
  }
}

```